client/ociindex/ociindex.go

```diff
@@ -15,6 +17,8 @@ import (
 const (
 	// lockFileSuffix is the suffix of the lock file
 	lockFileSuffix = ".lock"
+
+	annotationImageName = "io.containerd.image.name"
```

Comment:
This constant can be used https://github.com/containerd/containerd/blob/v2.0.1/core/images/annotations.go#L22

Reply:
I don't want to create a containerd dependency to this well-contained utility pkg.